noxfile.py

```diff
@@ -92,6 +94,35 @@ def bencher_baseline(session: nox.Session) -> None:
         silent=True,
     )
 
+@nox.session(python=["3.10", "3.11"], requires=["benchmark_model-{python}"])
+def bencher_feature_branch(session: nox.Session) -> None:
+    """
+    Run pytest benchmarks and upload them using Bencher (https://bencher.dev/) (cloud or self-hosted).
+    This session compares the performance of the feature branch with the historical baseline (as built from bencher_baseline session).
+    Alerts are raised if the performance of the feature branch is worse than the historical baseline (according to the thresholds).
+    """
+    session.run(
+        *f"bencher run \
+        --start-point main \
+        --start-point-clone-thresholds \
+        --start-point-reset \
+        --err \
+        --github-actions {os.environ['GD_COMMENT_TOKEN']} \
+        --ci-number {os.environ['PR_ID']} \
+        --ci-id run-{os.environ['BENCHER_TESTBED'].replace(":", "_")}-{int(datetime.now().strftime('%Y%m%d%H%M%S%f'))} \
```

Comment:
The double quotes are interpreted as the end of the main f-string and thus break the code.
```suggestion
        --ci-id run-{os.environ['BENCHER_TESTBED'].replace(':', '_')}-{int(datetime.now().strftime('%Y%m%d%H%M%S%f'))} \
```